backend/swagger/cirrusspark.yaml

```diff
@@ -0,0 +1,1257 @@
+swagger: "2.0"
+info:
+  description: "This is the REST diagram of Ciruss Spark API"
+  version: "1.0.0"
+  title: "Ciruss Spark"
+  termsOfService: "http://swagger.io/terms/"
+  contact:
+    email: "badobtech@gmail.com"
+  license:
+    name: "Apache 2.0"
+    url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+host: "cirrus-spark-backend-490.herokuapp.com"
+basePath: "/"
+tags:
+- name: "accounts"
+  description: "API for managing accounts"
+- name: "calls"
+  description: "API for managing calls"
+- name: "loghours"
+  description: "API for managing loghours"
+- name: "project"
+  description: "API for managing projects"
+- name: "tasks"
+  description: "API for managing tasks"
+- name: "auth"
+  description: "API for authorization"
+- name: "pet"
+  description: "Everything about your Pets"
+  externalDocs:
+    description: "Find out more"
+    url: "http://swagger.io"
+- name: "store"
+  description: "Access to Petstore orders"
```

Comment:
is this part of the template you used?

Reply:
Yes, we kept the pet endpoint as an example. We can remove it once we are done creating the Swagger